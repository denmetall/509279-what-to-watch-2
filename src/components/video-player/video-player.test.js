import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player';
import filmsMock from '../../mocks/films';
import createNodeMock from '../../mocks/create-node-mock';

it(`Components VideoPlayer renders correctly`, () => {
  const options = {createNodeMock};
  const tree = renderer
    .create(<VideoPlayer
      poster={filmsMock[0].previewImage}
      src={filmsMock[0].previewVideoLink}
      isPlaying={false}
    />, options)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
