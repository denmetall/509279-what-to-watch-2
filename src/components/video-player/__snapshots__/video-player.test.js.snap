// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components VideoPlayer renders correctly 1`] = `
<video
  height="175"
  muted={true}
  poster="img/johnny-english.jpg"
  width="280"
>
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  />
</video>
`;
